src/constant/generic.ts

```diff
@@ -8,7 +8,7 @@ import { Blockchain, BlockchainV2, EVMChain as EVMChainV2 } from "@carbon-sdk/ut
 export const DEFAULT_FEE_DENOM = "swth";
 
 // 0.0000001 SWTH
-// 100 SWTH sats
```

Comment:
I'm not quite sure how sats work. Maybe can remove this comment since we're not sure if this is correct.